include/aspect/material_model/entropy_model.h

```diff
@@ -117,6 +117,9 @@ namespace aspect
         double angle_of_internal_friction;
         double cohesion;
 
+        double multicomponent_max_iteration;
+        double multicomponent_tolerance;
```

Comment:
Add documentation to these parameters (you can use the same text as for the input parameters)